samples/MortgageApplication/application-conf/application.properties

```
@@ -44,6 +44,17 @@ excludeFileList=.*,**/.*,**/*.properties,**/*.xml,**/*.groovy,**/*.json,**/*.md,
 # sample: skipImpactCalculationList=**/epsmtout.cpy,**/centralCopybooks/*.cpy
 skipImpactCalculationList=
 
+# Flag to expand impact analysis to automatically include all mandatory submodules to build list in pipeline builds with a DBB Metadatastore connection based on the intermediate build list after impact analysis
+# This functionality queries and relies on the DBB Output collection of the application
+# 
+# The feature is specifically useful for applications using static calls to increase the isolation of the build environment
+# If the property is set to true, if a main program undergoes a change, all statically called sub programs are recompiled
+# Default: false
+# This property is setup as a DBB file property. This allows to enable the feature just for a subset of your build files
+# Sample to inspect the intermediate build list add submodules for Cobol build files
+# addSubmodulesToBuildList=true :: **/*.cbl
+addSubmodulesToBuildList=false :: **/*.cbl
```

This is a nice feature that can be show-cased by the MortgageApplication. Can we enable it for the MortgageApplication?

```suggestion
addSubmodulesToBuildList=true :: **/*.cbl
```